build/workbox.config.json

```diff
@@ -0,0 +1,15 @@
+{
+    "globDirectory": "./",
+    "globPatterns": [
+        "assets/**/*.{js,svg,png,css}",
+        "dist/**/*.{js,css}",
+        "docs/**/*.{md,html,css,jpg,js}"
+    ],
+    "swSrc": "./sw.js",
+    "swDest": "./_gh_pages/sw.js",
+    "templatedUrls": {
+      "/": ["_gh_pages/docs/index.html"],
+      "docs": ["_gh_pages/docs/index.html"],
+      "docs/4.0/": ["_gh_pages/docs/4.0/index.html"] 
```

Comment:
Why do we need this? If we cache the whole _gh_pages folder, doesn't workbox take care of this for us?